### ISSUE TITLE
Doesn't connect to friend's hosted server

### ISSUE THREAD

**Issue by Raxyz0** [TLeonardUK/ds3os#125]
Trying to join a private session always returns "Failed to log in to the DARK SOULS III game server."

- Tried hosting on both ends, the host can join it's own server, the client cannot and doesn't even log an connection attempt
- Double checked public and private IPs
- Double checked port forwading settings for both
- Shared the "server.ds3osconfig" file
- Put Loader and Server in firewall exceptions
- Ran in admin mode
- Ran without AV software
- Ran with Cloudflare Warp
- Repaired the VC Redist prerequisite

None of these had any noticeable effect.

**Comment by Celdath** [comment]
Did you find a fix to this?

**Comment by Raxyz0** [comment]
Nope, been trying for days. Still no dice. I'm considering using an older version but idk any more at this point.

**Comment by Cloudef** [comment]
Maybe your friend's ISP is blocking hosting on that particular port?

**Comment by ClimacticCat** [comment]
Getting the same issue over here. Tried most of the above to no avail. Not entirely sure how to check my ports though.

**Comment by Cloudef** [comment]
https://portchecker.co/ You can use online service like this too.

**Comment by Raxyz0** [comment]
Portchecker says the port is closed, but it says ALL ports are closed even email protocols that should be open since they're working fine. I'm gonna try and set up Netcat to see if it works better

**Comment by ClimacticCat** [comment]
Hi hi! Just coming back to update, my blind self didn't scroll down enough to read that some ports need to be forwarded for server hosting. Apologies.
